mail_tracking/models/ir_mail_server.py

```diff
@@ -79,7 +79,7 @@ def send_email(self, message, mail_server_id=None, smtp_server=None,
                 smtp_server=smtp_server, smtp_port=smtp_port,
                 smtp_user=smtp_user, smtp_password=smtp_password,
                 smtp_encryption=smtp_encryption, smtp_debug=smtp_debug)
-        except Exception as e:
+        except Exception as e:  # pragma: no cover
             if tracking_email:
                 tracking_email.smtp_error(self, smtp_server_used, e)
             raise
```

Comment:
I took a look at your mocks, and I do not see a test case for an SMTP Exception. In order to test this case, you need to use `mock_object.side_effect = SMTPError`. I only see return_values being set in the mocks.